### PR TITLE
Products

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Comment by MuskanChhatrasal** [comment]
Live Preview:  https://deploy-preview-2--thirsty-ramanujan-1a03a8.netlify.app/products

**Review by aayushjain-code — COMMENTED** [review]
Reduce the content it will be cleaner.
try to give single import file by clubbing components in one point.

**Review by aayushjain-code — APPROVED** [review]

File: src/Components/Navbar/navbar.js [hunk 1]
```diff
@@ -16,15 +25,18 @@ const Navbar = () => {
         <Link to='/products'><a>products</a></Link>
         <Link to='/wishlist'><a>Wishlist</a></Link>
         <Link to='/cart'><a>Cart</a></Link>
```
Comment (Developers-codz) [inline src/Components/Navbar/navbar.js:27]: Why `<a>` inside link ??

File: src/Components/Navbar/navbar.js [hunk 2]
```diff
@@ -16,15 +25,18 @@ const Navbar = () => {
...
       <div className="icons">
         <div className="fas fa-bars" id="menu-btn"></div>
-        <Link to='/cart'><div className="fas fa-shopping-cart" id="cart-btn">
+        <Link to='/cart'><div className="fas fa-shopping-cart" id="cart-btn" style={{position: 'relative'}}>
+        <span style={{position: 'absolute', top: -20, right: 0, fontSize: '1.5rem'}}>{cartQuantity}</span>
         </div></Link>
-        <div className="fas fa-user" id="login-btn">
-        </div>
+        <button className='loginButton'>Login In</button>
+        {/* <div className="fas fa-user" id="login-btn">
```
Comment (Developers-codz) [inline src/Components/Navbar/navbar.js:38]: remove commented code

File: src/index.js [hunk 3]
```diff
@@ -3,15 +3,19 @@ import ReactDOM from "react-dom";
...
 import {BrowserRouter} from 'react-router-dom'
+import { Provider } from "./Context/context";
+
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
+  <Provider>
```
Comment (Developers-codz) [inline src/index.js:14]: here, I think you can provide a better name like if its product provider then `ProductProvider`

**Review by Developers-codz — APPROVED** [review]
Everything else looks good to me

File: src/Components/Navbar/navbar.js (hunk 1 above)
Comment (MuskanChhatrasal) [reply src/Components/Navbar/navbar.js:27]: By mistake will correct that. Thankyou!!

File: src/Components/Navbar/navbar.js (hunk 2 above)
Comment (MuskanChhatrasal) [reply src/Components/Navbar/navbar.js:38]: Okayy!! Will do that

File: src/index.js (hunk 3 above)
Comment (MuskanChhatrasal) [reply src/index.js:14]: Will change that.

**Comment by MuskanChhatrasal** [comment]
> Reduce the content it will be cleaner. try to give single import file by clubbing components in one point.

Okayy!! Will do that. Thankyou

**Pull request merged**